src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/StrandBiasTest.java

```diff
@@ -0,0 +1,190 @@
+                final MostLikelyAllele mostLikelyAllele = PerReadAlleleLikelihoodMap.getMostLikelyAllele(el.getValue());
+                final GATKRead read = el.getKey();
+                updateTable(myTable, mostLikelyAllele.getAlleleIfInformative(), read, ref, allAlts);
+            }
+            if ( passesMinimumThreshold(myTable, minCount) ) {
+                copyToMainTable(myTable, table);
+            }
+        }
+
+        return table;
+    }
+
+    /**
+     * Helper method to copy the per-sample table to the main table
+     *
+     * @param perSampleTable   per-sample table (single dimension)
+     * @param mainTable        main table (two dimensions)
+     */
+    private static void copyToMainTable(final int[] perSampleTable, final int[][] mainTable) {
+        mainTable[0][0] += perSampleTable[0];
+        mainTable[0][1] += perSampleTable[1];
+        mainTable[1][0] += perSampleTable[2];
+        mainTable[1][1] += perSampleTable[3];
+    }
+
+    private static void updateTable(final int[] table, final Allele allele, final GATKRead read, final Allele ref, final List<Allele> allAlts) {
+        final boolean matchesRef = allele.equals(ref, true);
+        final boolean matchesAnyAlt = allAlts.contains(allele);
+
+        if ( matchesRef || matchesAnyAlt ) {
+            final int offset = matchesRef ? 0 : ARRAY_DIM;
+
+            // a normal read with an actual strand
+            final boolean isFW = !read.isReverseStrand();
+            table[offset + (isFW ? 0 : 1)]++;
+        }
+    }
+
+    /**
+     * Does this strand data array pass the minimum threshold for inclusion?
+     *
+     * @param data  the array
+     * @param minCount The minimum threshold of counts in the array
+     * @return true if it passes the minimum threshold, false otherwise
+     */
+    protected static boolean passesMinimumThreshold(final int[] data, final int minCount) {
+        // the ref and alt totals must be greater than MIN_COUNT
+        return data[0] + data[1] + data[2] + data[3] > minCount;
+    }
+
+    /**
+     * Helper function to parse the genotype annotation into the SB annotation array
+     * @param string the string that is returned by genotype.getAnnotation("SB")
+     * @return the array used by the per-sample Strand Bias annotation
+     */
+    private static int[] encodeSBBS( final String string ) {
+        final int[] array = new int[ARRAY_SIZE];
+        final StringTokenizer tokenizer = new StringTokenizer(string, ",", false);
+        for( int index = 0; index < ARRAY_SIZE; index++ ) {
+            array[index] = Integer.parseInt(tokenizer.nextToken());
+        }
+        return array;
+    }
+
+    /**
+     * Helper function to turn the  SB annotation array into a contingency table
+     * @param array the array used by the per-sample Strand Bias annotation
+     * @return the table used by the StrandOddsRatio annotation
```

Comment:
Oops -- this comment never got updated when I refactored this code.  Should return the table used by the StrandBiasTest annotation.